docker-entrypoint.sh

```diff
@@ -50,14 +78,64 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		done
 		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			      {loopback_users, []}
+		EOF
+
+		if [ "$RABBITMQ_MANAGEMENT_ENABLED" ]; then
+			cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
+				    ]
+				  },
+				  { rabbitmq_management, [
+				      { listener, [
+			EOF
+
+			if [ "${use_ssl}" = "yes" ]; then
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				      { port, 15671 }, 
+				      { ssl, true },
+				      { ssl_opts, [ 
+				          { certfile,   "$RABBITMQ_SSL_CERT_FILE" },
+				          { keyfile,    "$RABBITMQ_SSL_KEY_FILE" },
+				          { cacertfile, "$RABBITMQ_SSL_CA_FILE" },
+				      { verify,   verify_none },
+				      { fail_if_no_peer_cert, false } ] } ] }
+				EOS
+			else
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				        { port, 15672 },
+				        { ssl, false } 
+				        ]
+				      }
+				EOS
+			fi
+		fi
+
+		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			    ]
 			  }
 			].
 		EOF
 	fi
 
+	if [ "${use_ssl}" = "yes" ]; then
+		# Create combined cert
+		cat ${RABBITMQ_SSL_CERT_FILE} ${RABBITMQ_SSL_KEY_FILE} > /tmp/combined.pem
```

Comment:
These should be quoted, ie `cat "$RABBITMQ_SSL_CERT_FILE" "$RABBITMQ_SSL_KEY_FILE" > ...`.